src/hangman/events/handlers/new-game-started.handler.ts

```diff
@@ -1,13 +1,13 @@
 import { EventsHandler, IEventHandler } from '@nestjs/cqrs';
-import { NewGameStartedEvent } from '../Events/NewGameStarted.event';
 import { Logger } from '@nestjs/common';
-import { NewGameStartedUpdater } from '../Updaters/NewGameStarted.updater';
+import { NewGameStartedEvent } from '../impl/new-game-started.event';
+import { NewGameStartedUpdater } from '../updaters/new-game-started.updater';
 
 @EventsHandler(NewGameStartedEvent)
-export class NewGameStartedEventHandler
+export class NewGameStartedHandler
```

Comment:
I'd like to keep the naming as clear as possible. So although the name `NewGameStarted` sounds like an event already (and every event should have a name like that) still I think it's nice to keep it `NewGameStartedEventHandler`